File: AircraftWar/src/main/scala/work/chiro/game/bullet/EnemyBullet.scala
package work.chiro.game.bullet

import work.chiro.game.animate.AnimateContainer
import work.chiro.game.application.ImageResource
import work.chiro.game.basic.PositionType.Position
import work.chiro.game.basic.Vec2Double

/**
 * @author chiro2001
 */
class EnemyBullet(posInit: Position, animateContainer: AnimateContainer[Vec2Double], power: Int)
  extends AbstractBullet(posInit, animateContainer, power)
  with ImageResource {
  override def getImageCachedPath = "images/bullet_enemy.png"
}

File: core/src/main/java/work/chiro/game/bullet/EnemyBullet.java
package work.chiro.game.bullet;

import work.chiro.game.animate.AnimateContainer;
import work.chiro.game.config.AbstractConfig;
import work.chiro.game.vector.Vec2;

/**
 * @author hitsz
 */
public class EnemyBullet extends BaseBullet {
    public EnemyBullet(Vec2 posInit, AnimateContainer animateContainer, int power) {
        super(posInit, animateContainer, power);
    }
}
